### PR TITLE
Checks if member is on legacy plan and warns them if they are

### PR DESCRIPTION
**Pull request by itstehkman**
@djpetti is this ok? I changed the redirect to self.redirect("/"), incase you're working on localhost, so it just goes to root. I also changed the inheritance of the class to ProjectHandler so it could handle the base.html template through self.render.

- Kareem

### CODE REVIEW

File: billing.py [hunk 1]
```diff
@@ -5,18 +5,28 @@
 import webapp2
 
 from membership import Membership
+from plans import Plan
+from google.appengine.ext.webapp import template
```
Comment (djpetti) [inline billing.py]: These should be organized in alphabetical order, and the GAE SDK one should be in a different section from the local ones, separated by vertical whitespace.

File: billing.py [hunk 2]
```diff
@@ -5,18 +5,28 @@
...
+from project_handler import ProjectHandler
 
-class BillingHandler(webapp2.RequestHandler):
+import logging
```
Comment (djpetti) [inline billing.py]: I know not everyone does this, but in our code, imports generally go only at the top.

File: billing.py [hunk 3]
```diff
@@ -5,18 +5,28 @@
...
       # User is not (yet) a member.
-      self.redirect("http://signup.hackerdojo.com")
+      self.redirect("/")
     else:
       # Open billing information.
       url = member.spreedly_url()
-      self.redirect(url)
+      plan = Plan.get_by_name(member.plan)
+      if plan.legacy:
+        self.response.out.write(self.render("templates/billing_popup.html", url=url))
```
Comment (djpetti) [inline billing.py]: Line length... I'm enforcing the standard 80-character limit for Python code.

**Comment by djpetti** [comment]
So, as you can see, I'm being a little bit anal about style things.

For future reference, my goal is to make all the code compliant with [this](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Indentation) style guide.
Obviously, a lot of old stuff is not compliant, so the best I can do is try to fix as much as possible, and in the mean time, ensure that all new code is compliant.

Also, your commit message should be in the present tense.

**Comment by itstehkman** [comment]
@djpetti fixed

File: billing.py [hunk 4]
```diff
@@ -5,18 +5,26 @@
...
-      self.redirect("http://signup.hackerdojo.com")
+      self.redirect("/")
     else:
       # Open billing information.
       url = member.spreedly_url()
-      self.redirect(url)
+      plan = Plan.get_by_name(member.plan)
+      if plan.legacy:
+        self.response.out.write(self.render(
+          "templates/billing_popup.html", url=url))
```
Comment (djpetti) [inline billing.py]: This should be a 4-space indent, sense you're breaking a single line of code across multiple lines.

**Comment by djpetti** [comment]
This is really close...

**Comment by itstehkman** [comment]
@djpetti

**Comment by djpetti** [comment]
Okay, everything looks good. Can you squash those top two commits down into the first one? The way you might do this is using interactive rebase. For instance, you might do `git rebase -i HEAD~3`, and then change "pick" to "s" for the second two of your commits. Then you can push the whole thing up here again. (You'll probably have to do `git push -f`.)

**Comment by itstehkman** [comment]
@djpetti

**Comment by djpetti** [comment]
LGTM

**Pull request merged**